test/functional/cmdLineTests/criu/src/org/openj9/criu/OptionsFileTest.java

```diff
@@ -109,7 +109,7 @@ static void propertiesTest2() {
 	}
 
 	static void propertiesTest3() {
-		String optionsContents = "-Dprop1=val1\n-Dprop2=\\\nval2\n-Dprop3=v \\ \n a \\     \n  l3";
```

Comment:
this string was constructed this way to test cases where we need to trim between lines

Reply:
It seems only the leading spaces are ignored.
https://github.com/eclipse-openj9/openj9/blob/85fe1846d45b19b3f2ffaa6afc06609f9ad838c3/runtime/util_core/vmargs_core.c#L152
Replaced JDK `options.default` with the content `-Dprop1=val1\n-Dprop2=\\\nval2\n-Dprop3=v \\ \n a \\     \n  l3`, JVM failed to start.
```
JVMJ9VM007E Command-line option unrecognised: a
Error: Could not create the Java Virtual Machine.
```
On the other hand, JVM starts successfully with `-Dprop1=val1\n-Dprop2=\\\nval2\n-Dprop3=v \\\n a \\\n  l3` within the `options.default`.